src/main/graph_html.ts

```diff
@@ -11,13 +11,19 @@ const LEFT_DELTA = 170;
 const COLS = 4
 let nnodes = 0
 
+let vCoordUnconnectedNode = {
+  coord: -30,
+};
+export { vCoordUnconnectedNode }
+
 function _nextNodeLocation(): { top: number, left: number } {
   let row = Math.floor(nnodes / COLS)
   let col = nnodes - row * COLS
   nnodes += 1
+  vCoordUnconnectedNode.coord = vCoordUnconnectedNode.coord + 100
   return {
-    top: TOP_INITIAL + row * TOP_DELTA,
-    left: LEFT_INITIAL + col * LEFT_DELTA,
+    top: vCoordUnconnectedNode.coord,
+    left: 900,
```

Comment:
Can you create named constants for these numbers such as `100`, `900`, `-30`?

Reply:
Also, at `900`, unconnected nodes get covered by the sidebar on my screen unless my browser is at its maximum width. Could we put the unconnected nodes on the left side or something else?